package.json

```diff
@@ -1,10 +1,11 @@
 {
   "name": "to-do-list",
   "version": "1.0.0",
-  "description": "",
+  "description": "This project is about simple to do list app using webpack.",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
   },
```

Comment:
- [ ] Kindly consider adding a start script here so that the application can easily be run using the npm start command. I know this should have been done already in the previous milestones. right now I cannot run your app with `npm run start` because you have not included them in your package.json file. I think you need to go back to the previous milestone lesson and add configure the webpack.
error in the terminal
- [ ] And because the project is not even running by this command `npm start`, I cant even review other requirements if you have implemented them or not. so kindly consider fixing this in order to proceed.
![qw](https://user-images.githubusercontent.com/96683843/224412437-3a1f7760-a2cd-4b41-a53d-d1ad5b1b2fec.jpg)